media-video/syncplay/syncplay-1.6.4a.ebuild

```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2020 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+
+inherit python-r1
+
+DESCRIPTION="Client/server to synchronize media playback"
+HOMEPAGE="https://syncplay.pl"
+SRC_URI="https://github.com/Syncplay/syncplay/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc64 x86"
```

Comment:
Please set *all* arches to testing when you bump a package. Here you let x86 on stable (instead of ~x86).
You can do this automatically with ekeyword '~all' foo.ebuild from package app-portage/gentoolkit.

Reply:
I updated my pull request to include this.